tap_stripe/schemas/subscription_items.json

```diff
@@ -26,6 +26,296 @@
       "format": "date-time"
```

Comment:
Can we add a reference document link?

Reply:
added the doc link in the comment 
https://github.com/singer-io/tap-stripe/pull/114#discussion_r777866475